azurecompute-arm/src/test/java/org/jclouds/azurecompute/arm/features/NetworkInterfaceCardApiLiveTest.java

```diff
@@ -66,8 +69,18 @@ public void setup() {
    public void deleteNetworkInterfaceCardResourceDoesNotExist() {
 
       final NetworkInterfaceCardApi nicApi = api.getNetworkInterfaceCardApi(resourcegroup);
-      boolean status = nicApi.delete(NETWORKINTERFACECARD_NAME);
-      assertFalse(status);
+      URI uri = nicApi.delete(NETWORKINTERFACECARD_NAME);
```

Comment:
We expect this test to return null, so just assert that the URI is actually null and forget about the rest..